### DISCUSSION TITLE
Progress Report 4–8 July

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#55] [Progress Report]
🚴🏻 **Progress**

- #42 
- #48 
- Began work on #45 

😖 **Regressions**

None

💬 **To Discuss**

- Data sources for time series (#44)

📆 **Next Week**

- #45

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- #42 
- Container images are now auto-deployed by VLab into the cluster
- Figured out network configuration inside the cluster so the containers can communicate

😖 **Regressions**

- #54 

💬 **To Discuss**

- None

📆 **Next Week**

- Work with VLab to expose the ui service via a URL
- Work on infrastructure for integration tests
